jstests/core/explain_use_backup_plan.js

```diff
@@ -0,0 +1,30 @@
+// Test that the explain will use backup plan if the original winning plan ran out of memory in the
+// "executionStats" mode
+// This test was designed to reproduce SERVER-32721"
+
+load("jstests/libs/analyze_plan.js");
```

Comment:
Please put the contents of this test in an immediately-invoked-function-expression (https://en.wikipedia.org/wiki/Immediately-invoked_function_expression)